doc/api/deprecations.md

```diff
@@ -206,11 +206,11 @@ See [`Intl.Segmenter`](https://github.com/tc39/proposal-intl-segmenter).
 <a id="DEP0018"></a>
 ### DEP0018: Unhandled promise rejections
 
-Type: Runtime
+Type: End-of-Life
 
-Unhandled promise rejections are deprecated. In the future, promise rejections
-that are not handled will terminate the Node.js process with a non-zero exit
-code.
+Any rejected promise that is unhandled and garbage collected is going to exit
```

Comment:
Mention that users can opt out of this behaviour with the env flag and warn against it?

(By the way if you prefer I can PR these changes into this branch)